docs/docs/spec.md

```diff
@@ -649,8 +649,8 @@ The same-type relation is *not* defined to be reflexive or transitive.
     > This is the second easy case: `A` never includes `null`.
 
 -   `A` has a [nullness-subtype-establishing path] to any type whose base type
-    is the same as the base type of `F`, and `F` does *not* have
-    [nullness operator] `MINUS_NULL`.
+    is the same as the base type of `F`, and there is *not* reason to be
+    [worried] that `F` has [nullness operator] `MINUS_NULL`.
```

Comment:
Should we add an unspecified type example to the non-normative example below?

Reply:
I see that we don't even have a `MINUS_NULL` example right now. I've added one and then said some things about unspecified nullness. I'm not sure if I picked good things to say or not.